### PR TITLE
Iow 284

### PR DESCRIPTION
**Pull request by ssoper-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -178,8 +178,21 @@ stepFunctions:
...
+                  ResultPath: $.timeSeries
+                  TimeoutSeconds: 60
                   End: true
+                  Retry:
+                    - ErrorEquals:
+                      - States.TaskFailed
+                      IntervalSeconds: 35
+                      MaxAttempts: 2
+                      BackoffRate: 2
+                  Catch:
```
Comment (ayan-usgs) [inline serverless.yml:192]: Drop the error handler. If this fails, we're just restart the entire map state.

**Review by ayan-usgs — CHANGES_REQUESTED** [review]

File: serverless.yml (hunk 1 above)
Comment (ssoper-usgs) [reply serverless.yml:192]: dropped

File: serverless.yml [hunk 2]
```diff
@@ -178,8 +178,20 @@ stepFunctions:
...
+                    - ErrorEquals:
+                      - States.ALL
```
Comment (dsteinich) [inline serverless.yml:194]: does this need 
                      Next: defaultMapFail
                  Next: defaultMapSuccess
to cover the Catch's next step and the Task's next step?

Comment (ssoper-usgs) [reply serverless.yml:194]: @ayan-usgs maybe?

Comment (ayan-usgs) [reply serverless.yml:194]: I would suggest `Next: defaultMapFail` for the catch.

Comment (ssoper-usgs) [reply serverless.yml:194]: added

**Pull request merged**